custom/motorcontrollers/CANTalonFX.java

```diff
@@ -1,12 +1,12 @@
 package org.usfirst.frc4904.standard.custom.motorcontrollers;
 
-import com.ctre.phoenix.motorcontrol.InvertType;
-import com.ctre.phoenix.motorcontrol.NeutralMode;
-import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import com.ctre.phoenix6.signals.ForwardLimitValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import com.ctre.phoenix6.signals.ReverseLimitValue;
+import com.ctre.phoenix6.hardware.TalonFX;
 
-public class CANTalonFX extends WPI_TalonFX implements TalonMotorController {
-	protected static final NeutralMode DEFAULT_NEUTRAL_MODE 	= NeutralMode.Coast;
-	protected static final InvertType  DEFAULT_INVERT_TYPE  	= InvertType.FollowMaster;
+public class CANTalonFX extends TalonFX {
+	protected static final NeutralModeValue DEFAULT_NEUTRAL_MODE 	= NeutralModeValue.Coast;
```

Comment:
should change that comment block, inverted is not a thing for the falcons here anymore i don't think